src/context-selection/use-deselect/use-deselect-on-coc-change.js

```diff
@@ -0,0 +1,15 @@
+import { useEffect, useState } from 'react'
+import { useCategoryOptionComboSelection } from '../use-context-selection.js'
+
+export default function useDeselectOnCocChange(deselectAll) {
+    const [initialExecution, setInitialExecution] = useState(true)
+    const [categoryOptionComboSelection] = useCategoryOptionComboSelection()
+
+    useEffect(() => {
+        if (!initialExecution) {
```

Comment:
Don't you want the opposite here? `if (initialExecution)`. Same for the other hooks on this level.

Reply:
The initial/default value is `true`